crd/apis/network/v1alpha1/gkenetworkparamset_types.go

```diff
@@ -70,9 +70,6 @@ type GKENetworkParamSetStatus struct {
 	// +optional
 	PodCIDRs *NetworkRanges `json:"podCIDRs,omitempty"`
 
-	// Conditions is a field representing the current conditions of the GKENetworkParamSet.
-	Conditions []metav1.Condition `json:"conditions"`
```

Comment:
since this was already merge, lets not risk it by remove things. We should not add new things here, but lets not remove anything, because this breaks compatibility.